lib/compiler-singlepass/src/codegen.rs

```diff
@@ -292,7 +319,7 @@ impl<'a, M: Machine> FuncGen<'a, M> {
 
         for (loc, _) in locs.iter().rev() {
             if let Location::Memory(..) = *loc {
-                self.check_location_on_stack(loc, stack_offset)?;
+                self.check_location_on_stack(loc, stack_offset).unwrap();
```

Comment:
This should be unwrap or `?` ?

Reply:
Technically, that's an internal error (bug), but yes, let's use `?`.